src/azure-cli/azure/cli/command_modules/iot/tests/latest/test_iot_commands.py

```diff
@@ -395,6 +395,124 @@ def test_iot_hub(self, resource_group, resource_group_location, storage_account)
         # Test 'az iot hub delete'
         self.cmd('iot hub delete -n {0}'.format(hub), checks=self.is_empty())
 
+
+    @AllowLargeResponse()
+    @ResourceGroupPreparer(location='westus2')
+    @StorageAccountPreparer()
+    def test_identity_hub(self, resource_group, resource_group_location, storage_account):
+        # Test IoT Hub create with identity
+        subscription_id = self._get_current_subscription()
+        rg = resource_group
+        location = resource_group_location
+        
+        identity_hub = 'iot-cli-identity-enabled-hub-test'
+        identity_based_auth = 'identityBased'
+        key_based_auth = 'keyBased'
+        event_hub_identity_endpoint_name = 'EventHubIdentityEndpoint'
+
+        containerName = 'iothubcontainer1'
+        storageConnectionString = self._get_azurestorage_connectionstring(rg, containerName, storage_account)
+        endpoint_name = 'Event1'
+        endpoint_type = 'EventHub'
+        eventhub_endpoint_uri = 'eh://test'
+        storage_cs_pattern = 'DefaultEndpointsProtocol=https;EndpointSuffix=core.windows.net;AccountName='
+        entity_path = 'entity'
+        
+        #identity hub creation
+        import os
+        templateFile = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'template.json')
+        self.cmd('deployment group create --name {0} -g {1} --template-file "{2}" --parameters name={3} --parameters location={4}'.format("identity-hub-deployment", resource_group, templateFile, identity_hub, location))
```

Comment:
This is the temporary test workaround you were describing while you wait for the SDK right?

Reply:
Yes - in order to create a hub with the correct properties in the correct SU, this is the ARM template used.